pontoon/administration/management/commands/import_verbatim.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from django.contrib.auth.models import Permission, User
+from django.core.management.base import (
+    BaseCommand,
+    CommandError,
+    NoArgsCommand,
+)
+from optparse import make_option
+from pontoon.base.models import Translation
+
+import urllib2
+import json
+
+
+class Command(BaseCommand):
+    help = 'Import translation authors and dates from Verbatim.'
+
+    option_list = NoArgsCommand.option_list + (
+        make_option('--project', action='store', dest='project',
+                    help='Project slug to import Verbatim data to'),
+        make_option('--locale', action='store', dest='locale',
+                    help='Locale code to import Verbatim data to'),
+        make_option('--filename', action='store', dest='filename',
+                    help='File name to import Verbatim data from'),
+        )
+
+    def handle(self, *args, **options):
+        project = options.get('project', False)
+        locale = options.get('locale', False)
+        filename = options.get('filename', False)
+
+        if not project or not locale or not filename:
```

Comment:
I'm not sure we care, but Django 1.8 has a new way of adding arguments that might make this particular line unnecessary: https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#accepting-optional-arguments